Packs/CortexAttackSurfaceManagement/Scripts/GetProjectOwners/GetProjectOwners.py

```diff
@@ -0,0 +1,100 @@
+import demistomock as demisto  # noqa: F401
+from CommonServerPython import *  # noqa: F401
+"""Script for identifying and recommending the most likely owners of a discovered service
+from those surfaced by Cortex ASM Enrichment.
+"""
+
+
+from typing import Dict, List, Any
+import traceback
+import re
+
+
+def is_gcp_iam_account(service_account: str) -> bool:
+    """
+    Determine whether service account is user-managed
+
+    Docs: https://cloud.google.com/iam/docs/service-account-types
+    """
+    return bool(re.search("^.+@[^\.]+(\.iam\.gserviceaccount\.com)$", service_account))
+
+
+def extract_project_name(service_account: str) -> str:
+    """
+    Extract project name from GCP IAM service account
+    """
+    match = re.search(r"(?<=@)[^\.]+(?=\.iam\.gserviceaccount\.com)", service_account)
+    if match:
+        return match.group()
+    else:
+        raise ValueError(f"Could not extract project name from service account {service_account}")
+
+
+def get_iam_policy(project_name: str) -> List[Dict[str, Any]]:
+    """
+    Retrieve IAM policy for project
+    """
+    try:
+        return demisto.executeCommand("gcp-iam-project-iam-policy-get", args={"project_name": f"projects/{project_name}"})
```

Comment:
Check for if the integration with `gcp-iam-project-iam-policy-get` is available either in the script or in the playbook. If you do it in a script it requires `demisto.getModules()` and getting the `state` and `brand`.

Reply:
tested that the error handling in main works! if there are GCP owners in the unranked list but the integration is disabled, it logs "No additional owners found" and continues the playbook